sklearn/metrics/_ranking.py

```diff
@@ -965,6 +971,8 @@ def precision_recall_curve(
 
     Examples
     --------
+    >>> # positive class (y_true = 1) is positively correlated with y_score and negative
+    >>> # class is negatively correlated with y_score
```

Comment:
I don't think this is needed, if we've already said in the main docstring that the 'default' assumption is that the positive class is positively correlated with `y_score`